### ISSUE TITLE
Change dynamically the url, is it possible?

### ISSUE THREAD

**Issue by lnkpaulo** [perliedman/leaflet-realtime#15]
First thanks for this superb plugin.

My question, is it possible to change the url dynamically when the loop is running?

E.g. the plugin starts with something like:

```javascript
...
url: '/api/vehicle/1'
...
```
And then with a click event changing it to:

```javascript
...
url: '/api/vehicle/2'
...
```

during the loop execution set by interval.

Thanks for the support.

**Comment by perliedman** [comment]
Actually not at this point, although it seems like a valid thing to support.

As a hack/workaround, you should be able directly set the `_src` property of the layer. Having said that, note that it _is_ a hack, since you shouldn't really modify things starting with underscore.

I'll keep this issue open, in the long run we should add a `setUrl` method or similar. A pull request would be most welcome!

**Comment by lnkpaulo** [comment]
Thanks for the feedback and the workaround.

Let me dig into the code and in case of success will be a pleasure make a pull request.

**Comment by perliedman** [comment]
Closed by 12a3ab041103ed1d154fdb1557652a681bf7e9a3.
